packages/cli/src/services/util.ts

```diff
@@ -43,6 +43,35 @@ export async function walkDirectory (
   }
 }
 
+export function findFilesRecursively (directory: string, ignoredPaths: Array<string> = []) {
```

Comment:
Don't we have a directory walker somewhere? I think we need a more flexible solution to support multiple scenarios. This is already the case at the runners IMO

Reply:
We have `walkDirectory` here in `util.ts`. It's unused, though, and calls a callback rather than giving a list of matching files. I'll probably delete it.

`findFilesRecursively` should be basically the same as the one we have in runners, it's just missing the `filePredicate` arg. Should I go ahead and proactively add the `filePredicate` arg? Is there anything else that I should add to this one?